.github/workflows/ci.yml

```diff
@@ -0,0 +1,94 @@
+name: SROS2 CI
+on:
+  pull_request:
+  push:
+  schedule:
+    # Run daily
+    - cron:  '0 12 * * *'
+
+jobs:
+  build_and_test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+    steps:
+    - uses: ros-tooling/setup-ros@0.0.13
+    - uses: ros-tooling/action-ros-ci@0.0.13
```

Comment:
RfC: not sure how to manage action versions in workflows. Is there a way to bump those automatically? otherwise maybe we should just point at the `master` branch instead of a specific version?

Reply:
If I remember correctly you can leave off the version entirely, but I think it's probably smart to leave it pinned and update consciously or things will break on us and require a flag day to fix.

Reply:
Pointing directly to `master` opens the way for breakage, I think we better manually bump it as we see fit.

Reply:
sounds good, kept the pinned approach but bumped to latest release at https://github.com/ros2/sros2/pull/178/commits/3cb4573242b9ad304e69918591145369389f3ffe